### PR TITLE
Added a function for division 

### PR DESCRIPTION
**Pull request by mariya-mamajiwala**

### CODE REVIEW

File: divide.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def divide(x,z):
```
Comment (ns-rse) [inline divide.py]: I think there is a tpyo here and it should be...

```suggestion
def divide(x,y):
```

Or you could `return x/z`

**Review by ns-rse — CHANGES_REQUESTED** [review]
Thanks for the contribution @mariya-mamajiwala I noticed  a small tpyo in and have made a suggestion, are you able to fix this please?

File: divide.py (hunk 1 above)
Comment (mariya-mamajiwala) [reply divide.py]: Thanks @ns-rse, have now made the corrections as suggested.

**Review by ns-rse — COMMENTED** [review]
Great, thanks for fixing that so quickly @mariya-mamajiwala 

Unfortunately we have a merge conflict to resolve, I think @twinkarma is going to take us through that shortly.

File: tests/test_divide.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+rom pythoncalculator import divide
```
Comment (twinkarma) [inline tests/test_divide.py]: Missing the f from from 
```suggestion
from pythoncalculator import divide
```

File: divide.py
Comment (twinkarma) [inline divide.py:1]: This file shoud be inside the `pythoncalculator/` directory.

**Review by twinkarma — CHANGES_REQUESTED** [review]
Looking good, just a few suggestions to fix this!

**Review by ns-rse — APPROVED** [review]
Thanks for spotting those issues @twinkarma and thanks for fixing things so quickly @mariya-mamajiwala this looks good to merge.

**Review by twinkarma — APPROVED** [review]
Great, thanks for making the changes so quickly!

**Pull request merged**